dtests/src/test/java/io/snappydata/hydra/externalHiveMetaStore/hiveMetaStore.java

```diff
@@ -1,4 +1,418 @@
+            throw new TestException("Beeline : Exception in closing connection",se);
+        }
+    }
+
+    public static  void HydraTask_DisconnectSnappy(){
+        try {
+            metaStore.snappyConnection.close();
+            if(metaStore.snappyConnection.isClosed()) {
+                Log.getLogWriter().info("Snappy : Connection closed");
+            }
+        } catch(SQLException se) {
+            throw new TestException("Snappy : Exception in closing connection",se);
+        }
+    }
+
+    // Presently not in use - Experimental.
+    public static void  HydraTask_TestBeeline() {
+        try {
+            metaStore.st = metaStore.beelineConnection.createStatement();
+            metaStore.rs = metaStore.st.executeQuery(showTblsDefault);
+            while(metaStore.rs.next()) {
+                Log.getLogWriter().info("Beeline Result : " + metaStore.rs.getString(1));
+            }
+        }catch(SQLException se) {
+            throw new TestException("Beeline : Exception for show tables", se);
+        }
+    }
+
+    // Presently not in use  - Experimental.
+    public static void  HydraTask_TestSnappy() {
+        try {
+            metaStore.snappyConnection.createStatement().execute(setexternalHiveCatalog);
+            Log.getLogWriter().info("Pointing to external hive meta store");
+            metaStore.snappyST = metaStore.snappyConnection.createStatement();
+            Log.getLogWriter().info("create statement from snappy");
+            metaStore.snappyRS = metaStore.metaStore.snappyST.executeQuery(showTblsDefault);
+            Log.getLogWriter().info("execute show tables in default");
+            while(metaStore.snappyRS.next()) {
+                Log.getLogWriter().info("Snappy Result : " + metaStore.snappyRS.getString(1) + " | " + metaStore.snappyRS.getString(2 ));
+            }
+        }catch(SQLException se) {
+            throw new TestException("Snappy : Exception for show tables", se);
+        }
+    }
+
+    private static String createExternalSnappyTbl(String tableName, String filePath) {
+        return "create external table if not exists " + tableName + " using csv options(path '" + filePath + "',header 'true')";
+    }
+
+    private static String createSnappyTblAndLoadData(String tableName,String buckets,String stageTable) {
+        return "create table if not exists " + tableName + " using column options(BUCKETS '" + buckets + "') as select * from " + stageTable;
+    }
+
+    private static String createHiveTables(String tableDefinition) {
+        return "create table " + tableDefinition + " row format delimited fields terminated by ',' ";   //tblproperties(\"skip.header.line.count\"=\"1\")"
+    }
+
+    private static String loadDataToHiveTbls(String dataPath, String tblName) {
+        return "load data local inpath '" + dataPath + "' overwrite into table " + tblName;
+    }
+
+    private static void validationRoutine(String dir, ResultSet rs1, ResultSet rs2, String index) {
```

Comment:
Give an appropriate name to this method.

Reply:
Done